packages/analytics/src/integrations/Omnitracking/__tests__/omnitracking-helper.test.ts

```diff
@@ -65,7 +65,7 @@ describe('getPlatformSpecificParameters', () => {
 });
 
 describe('getCheckoutEventGenericProperties', () => {
-  it('should display some warn', () => {
+  it.skip('should display some warn', () => {
```

Comment:
Is this intended?

Reply:
The warn tested here was removed from the method so this test got no purpose, I will remove the test